### DISCUSSION TITLE
What settings are best paired with this shader?

### DISCUSSION THREAD

**Discussion by samlovesgithub** [jebbyk/OSBES-minecraft-bedrock-edition-shader#110] [Q&A] [answered]
Hey there! Quick question! What MCPE settings do your recommend being used with this shader? Thank you and have a great week!

**Comment by OpenSauce04** [comment, marked as answer]
Whatever works best for you, although I would recommend a decently high render distance (12+) if your phone can handle it, as the fog can look a bit too thick otherwise

**Comment by Andriusito** [comment]
I would like you to optimize the shader more, since the previous version is a little more fluid for me, besides that I would like you to remove the fog

**Comment by Andriusito** [comment]
I would like you to optimize the shader more, since the previous version is a little more fluid for me, besides that I would like you to remove the fog
